### ISSUE TITLE
Uncaught ReferenceError: getEditor is not defined

### ISSUE THREAD

**Issue by Synthetica9** [idris-hackers/atom-language-idris#150]
### Steps to reproduce:

Try to solve any hole.

**Atom Version**: 1.13.0
**Electron Version**: 1.4.14
**System**: linux 4.8.13-1-ARCH
**Thrown From**: [language-idris](https://github.com/idris-hackers/atom-language-idris) package, v0.4.7

### Stack Trace

Uncaught ReferenceError: getEditor is not defined

```
At /home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:78

ReferenceError: getEditor is not defined
    at IdrisController.doProofSearch (/home/synthetica/.atom/packages/language-idris/lib/idris-controller.coffee:344:14)
    at /home/synthetica/.atom/packages/language-idris/lib/idris-controller.coffee:1:1
    at AnonymousObserver._onNext (/home/synthetica/.atom/packages/language-idris/lib/idris-controller.coffee:80:11)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1635:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at AnonymousObserver.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at AutoDetachObserverPrototype.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6260:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at CatchObserver.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:2747:59)
    at CatchObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at CatchObserver.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at AutoDetachObserverPrototype.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6260:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at InnerObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:3263:90)
    at InnerObserver.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at AutoDetachObserverPrototype.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6260:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at scheduleItem (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:2651:16)
    at JustSink.run (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:2659:9)
    at JustObservable.subscribeCore (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:2640:19)
    at JustObservable.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at setDisposable [as action] (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1813:46)
    at ScheduledItem.invokeCore (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:858:33)
    at ScheduledItem.invoke (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:846:40)
    at runTrampoline (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1045:37)
    at tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at CurrentThreadScheduler.schedule (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1061:45)
    at JustObservable.Rx.ObservableBase.ObservableBase._subscribe (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1826:32)
    at JustObservable.Rx.Observable.observableProto.subscribe.observableProto.forEach (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1703:19)
    at MergeAllObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:3232:37)
    at MergeAllObserver.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at AutoDetachObserverPrototype.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6260:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at InnerObserver.Rx.FlatMapObservable.InnerObserver.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1876:14)
    at InnerObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at InnerObserver.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at AutoDetachObserverPrototype.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6260:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at InnerObserver.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:4329:14)
    at InnerObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at InnerObserver.tryCatcher (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:64:31)
    at AutoDetachObserverPrototype.next (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6260:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:1569:31)
    at AsyncSubject.onCompleted (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:6462:17)
    at handler (/home/synthetica/.atom/packages/language-idris/node_modules/rx-lite/rx.lite.js:4766:7)
    at FSReqWrap.oncomplete (fs.js:123:15)
```

### Commands

```
     -0:01.8.0 core:move-down (input.hidden-input)
  7x -0:01.4.0 core:move-right (input.hidden-input)
     -0:00.0 language-idris:proof-search (input.hidden-input)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "disabledPackages": [
      "markdown-preview",
      "autocomplete-julia",
      "fonts",
      "the-closer",
      "racer",
      "Clean2",
      "term",
      "run",
      "run-command",
      "term3",
      "sync-on-save",
      "intellij-idea-keymap",
      "atom-scrolloff",
      "susave",
      "tidy-markdown",
      "language-x86asm"
    ],
    "telemetryConsent": "no",
    "themes": [
      "atom-material-ui",
      "one-dark-syntax"
    ]
  },
  "language-idris": {}
}
```

### Installed Packages

```coffee
# User
2-dark-syntax, v0.1.1 (inactive)
aligner, v0.22.3 (active)
atom-beautify, v0.29.16 (inactive)
atom-html-preview, v0.1.22 (active)
atom-material-ui, v1.3.9 (active)
auto-indent, v0.5.0 (active)
autoclose-html, v0.23.0 (active)
autocomplete-clang, v0.10.0 (active)
autocomplete-haskell, v0.6.7 (inactive)
autocomplete-julia, v0.1.0 (inactive)
autocomplete-python, v1.8.26 (active)
cursor-history, v0.9.1 (active)
figlet, v0.5.4 (active)
file-icons, v2.0.9 (active)
fold-lines, v0.2.1 (active)
fold-navigator, v0.0.618 (inactive)
fonts, v2.0.0 (inactive)
glacier-dark-ui, v0.5.0 (inactive)
haskell-ghc-mod, v1.19.5 (inactive)
haskell-pointfree, v0.1.3 (active)
hex, v0.6.2 (active)
highlight-selected, v0.12.0 (active)
ide-haskell-repl, v0.5.1 (inactive)
indent-guide-improved, v1.4.12 (active)
language-assembly, v0.7.1 (active)
language-diff, v0.7.0 (active)
language-fish-shell, v1.0.5 (active)
language-haskell, v1.10.4 (active)
language-idris, v0.4.7 (active)
language-javascript-semantic, v0.2.1 (active)
language-julia, v0.10.0 (active)
language-latex, v1.0.0 (active)
language-llvm, v0.1.1 (active)
language-pfm, v0.82.0 (active)
language-prolog, v0.10.0 (active)
language-rust, v0.4.9 (active)
language-x86asm, v0.9.0 (inactive)
latex-completions, v0.3.2 (active)
line-count, v0.5.0 (inactive)
linter, v1.11.21 (active)
linter-gcc, v0.6.15 (active)
linter-hlint, v0.5.0 (active)
linter-pep8, v1.3.2 (active)
local-history, v4.0.1 (active)
MagicPython, v1.0.3 (active)
markdown-folder, v0.5.0 (inactive)
markdown-preview-plus, v2.4.0 (active)
markdown-preview-plus-opener, v0.2.0 (active)
markdown-scroll-sync, v2.1.2 (active)
markdown-table-formatter, v2.8.4 (active)
markdown-writer, v2.6.3 (inactive)
minimap, v4.26.6 (active)
minimap-cursorline, v0.2.0 (active)
minimap-find-and-replace, v4.5.1 (active)
minimap-git-diff, v4.3.1 (active)
minimap-highlight-selected, v4.5.0 (active)
minimap-linter, v1.6.0 (active)
minimap-pigments, v0.2.2 (active)
minimap-selection, v4.4.0 (active)
minimap-split-diff, v0.3.2 (active)
multi-cursor, v2.1.5 (inactive)
nucleus-dark-ui, v0.10.22 (inactive)
one-dark-mini-ui, v2.0.5 (inactive)
one-dark-vivid-syntax, v1.7.1 (inactive)
open-recent, v5.0.0 (active)
pigments, v0.38.0 (active)
python-indent, v1.0.0 (active)
python-tools, v0.6.9 (active)
racer, v0.20.0 (inactive)
scrolloff, v0.2.0 (active)
seti-ui, v1.4.0 (inactive)
split-diff, v1.1.1 (inactive)
susave, v0.4.0 (inactive)
sync-settings, v0.8.1 (active)
the-closer, v0.6.0 (inactive)
tidy-markdown, v3.0.1 (inactive)
todo-show, v1.9.0 (inactive)
tool-bar, v1.0.1 (active)
tool-bar-almighty, v0.8.1 (active)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (inactive)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (active)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.14.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.33.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.43.2 (active)
bracket-matcher, v0.85.1 (active)
command-palette, v0.39.1 (inactive)
deprecation-cop, v0.55.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.2.0 (active)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.1 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.159.1 (inactive)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.5 (active)
status-bar, v1.6.0 (active)
styleguide, v0.48.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (active)
timecop, v0.33.2 (active)
tree-view, v0.211.1 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.39.0 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.1 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by melted** [comment]
Oh dear, I forgot a '@'.

v0.4.8 coming soon.

**Comment by melted** [comment]
I have published v0.4.8 with the fix.

Thanks for the bug report!
